packages/ui/react-ui-thread/src/Message/Message.tsx

```diff
@@ -59,7 +59,7 @@ export const Message = ({ timestamp, children, ...messageMeta }: MessageProps) =
   return (
     <MessageMeta {...messageMeta} continues>
       <p className='grid grid-cols-[1fr_max-content] gap-2 pie-2'>
-        <Avatar.Label classNames={['truncate text-sm font-medium', !messageMeta.authorName && 'fg-description']}>
+        <Avatar.Label classNames={['truncate text-sm font-thin', !messageMeta.authorName && 'fg-description']}>
```

Comment:
```suggestion
        <Avatar.Label classNames={['truncate text-sm font-normal', !messageMeta.authorName && 'fg-description']}>
```
We should definitely not use `font-thin` and `text-sm` at the same time — no software in the ecosystem does this because it is very illegible.

Reply:
ok i'll try fg-subdued